README.MD

```diff
@@ -227,4 +234,4 @@ export class MyElement {
 ## More to come!
 Typed Web Components is in an early phase and needs your feedback. Please
 try it out and if you find an issue post it in issues tab. Also, do not
-hesitate to also post ideas!
\ No newline at end of file
+hesitate to also post ideas!
```

Comment:
Empty lines at the end :P. I usually keep 1 empty line at the end and it's usually enough to avoid that git diff issue.